tools/releases/dockerfiles/Dockerfile.kuma-init

```diff
@@ -1,13 +1,8 @@
 # When updatinng image version, remember to update it in tools/releases/distros.sh and enterprise version
 FROM kumahq/kuma-net-ebpf:0.8.10 as ebpf
-# until there is a distroless iptables image we have to use something else
-FROM ubuntu:jammy
+FROM gcr.io/k8s-staging-build-image/distroless-iptables:v0.1.2
```

Comment:
Sounds ok but who maintains this atm?

Reply:
It's Kubernetes https://github.com/kubernetes/release/tree/master/images/build/distroless-iptables